Header header
int32 id

geometry_msgs/Point position
geometry_msgs/Point velocity
float32 yaw

bool vo_available
int32[] dismap_ids
float64[] dismap_dists

node_detected_xyzyaw[] detected